hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -60,6 +65,8 @@
   protected long recordsDeleted = 0;
   private Map<String, HoodieRecord<T>> recordMap;
   private boolean useWriterSchema = false;
+  private HoodieTimer writeTimer = null;
```

Comment:
Used in all handles so maybe move it to the base class.

Reply:
Done.